File: SchoolApi/Models/Exam.cs
using System;

namespace SchoolApi.Models
{
    public class Exam
    {
        public int ID { get; set; }
        public int ClassroomID { get; set; }
        public int SubModuleID { get; set; }
        public DateTime From { get; set; }
        public DateTime To { get; set; }
        public string Description { get; set; }
        public Classroom Classroom { get; set; }
        public SubModule SubModule { get; set; }
    }
}

File: SchoolSpa/src/app/_models/exam.ts
import { Classroom } from "./classroom";
import { SubModule } from "./sub-module";

export interface Exam{
    id : number,
    classroomID : number,
    subModuleID : number,
    from : Date,
    to : Date,
    descriptions : string,
    classroom : Classroom,
    subModule : SubModule
}